tortoise/verifying_tortoise.go

```diff
@@ -36,11 +61,38 @@ func blockMapToArray(m map[types.BlockID]struct{}) []types.BlockID {
 type turtle struct {
 	logger log.Log
 
-	bdp blockDataProvider
+	atxdb atxDataProvider
+	bdp   blockDataProvider
+	clock layerClock
+
+	// TODO: can some or all of these be unexported? is this an issue for serialization?
```

Comment:
it is a serialization issue, w/e you want to be serialized (at least with current xdr impl) has to be exported.

Reply:
got it, I thought so, thanks. updated the comment accordingly.